lib/rspec/sidekiq/matchers/have_enqueued_sidekiq_job.rb

```diff
@@ -1,11 +1,11 @@
 module RSpec
   module Sidekiq
     module Matchers
-      def have_enqueued_job(*expected_arguments)
-        HaveEnqueuedJob.new expected_arguments
+      def have_enqueued_sidekiq_job(*expected_arguments)
+        HaveEnqueuedSidekiqJob.new expected_arguments
       end
```

Comment:
wouldn't how this works depend on gem load order (i.e. if rspec-rails is loaded before or after this gem)